sources/engine/Stride.Input/Windows/InputSourceWindowsDirectInput.cs

```diff
@@ -4,9 +4,11 @@
 #if (STRIDE_UI_WINFORMS || STRIDE_UI_WPF)
 using System;
 using System.Collections.Generic;
+using System.IO;
+using System.Management;
 using SharpDX;
 using SharpDX.DirectInput;
-using Stride.Native.DirectInput;
+using Microsoft.Win32;
```

Comment:
would that make a problem with linux migration?

Reply:
Note that this file compiles only when STRIDE_UI_WINFORMS or STRIDE_UI_WPF and are set only when StrideFramework is Windows

Reply:
so propably no